pyincore/client.py

```diff
@@ -261,6 +260,22 @@ def store_authorization_in_file(self, authorization: str):
         except IOError as e:
             logger.warning(e)
 
+    def is_token_expired(self, token):
+        """Check if the token has expired
+
+        Returns:
+             True if the token has expired, False otherwise
+        """
+        # Split the token to get payload
+        _, payload_encoded, _ = token.split('.')
+        # Decode the payload
+        payload = base64.urlsafe_b64decode(payload_encoded + '==')  # Padding just in case
+        payload_json = json.loads(payload)
+        now = datetime.now(timezone.utc)
+        current_time = now.timestamp()
+        # Compare current time with exp claim
```

Comment:
Do we want to add any debugging or info statement here, something like:

logger.info("Token expires on " + datetime.fromtimestamp(payload_json['exp']).strftime("%c"))

Reply:
This info will show in notebook when we call IncoreClient, and users don't know JWT tokens exist so this info may confuse them. This is my only concern

Reply:
Or debug, just thinking of ways if a power user wants to debug a token issue. It's fine without it.